gradle.properties

```
@@ -2,13 +2,16 @@ org.gradle.caching=true
 group=io.ballerina.stdlib
 version=2.15.1-SNAPSHOT
```

Let's bump to the next minor version with this.

```suggestion
version=2.16.0-SNAPSHOT
```